src/cachemachine/rubinimageinfo.py

```diff
@@ -0,0 +1,346 @@
+import re
+from dataclasses import dataclass
+from enum import Enum, auto
+from typing import Optional
+
+import semver
+from docker_image import reference
+
+from .types import IncomparableImagesError
+
+DEFAULT_TAG = "latest"  # Docker convention
+DEFAULT_RECOMMENDED_TAG = "recommended"
+
+
+class RubinImageType(Enum):
+    """Class to capture the different kinds of image types possible in a
+    Rubin Lab Image container, given our standard tag set."""
+
+    RECOMMENDED = auto()
+    LATEST = auto()
+    LATEST_SOMETYPE = auto()
```

Comment:
this is a really confusing thing.  and since we don't use latest weekly or latest daily right now, I would suggest we just not support this

Reply:
T&S do use them.